utils/meta.go

```diff
@@ -25,9 +25,6 @@ func NewValue(t reflect.Type) (v reflect.Value) {
 func ToArray(value interface{}) (values []string) {
 	switch value := value.(type) {
 	case []string:
-		if len(value) == 1 && value[0] == "" {
```

Comment:
This is required by select many, as html's select many widget will submit a array with a blank string when none data selected, in this case, we want a empty array, but not with a nil value.

 `[]string{""}` should return `[]string{}`, but not `[]string{nil}`